charts/swoop-caboose/README.md

```diff
@@ -132,6 +132,160 @@ time="2023-07-28T17:45:34Z" level=info msg="index config" indexWorkflowSemaphore
+export MINIO_SECRET_KEY=`helm get values minio -a -o json | jq -r .service.secretAccessKey | base64 --decode`
+mc alias set swoopminio http://127.0.0.1:9000 $MINIO_ACCESS_KEY $MINIO_SECRET_KEY
+```
+
+####  Run the [copy-assets-stac-task](https://github.com/Element84/copy-assets-stac-task) argo workflow example
+
+First clone the [copy-assets-stac-task](https://github.com/Element84/copy-assets-stac-task) repository.
+
+After cloning the [copy-assets-stac-task](https://github.com/Element84/copy-assets-stac-task) repository, first proceed to modify the `payload_workflow.json` and replace the `<REPLACE_WITH_ASSETS_S3_BUCKET_NAME>` with the bucket name created on the [Pre-requisites section](#pre-requisites-to-running-argo-workflows-example).
+
+Create a public minio local bucket and copy the `payload_workflow.json` file after replacing the `<REPLACE_WITH_ASSETS_S3_BUCKET_NAME>` name. To do this via the minio client:
+
+```
+$ mc mb swoopminio/payloadtest
+
+Bucket created successfully `swoopminio/payloadtest`.
+```
+```
+$ mc anonymous set public swoopminio/payloadtest
+
+Access permission for `swoopminio/payloadtest` is set to `public`
+```
+```
+$ mc cp ./payload_workflow.json swoopminio/payloadtest/
+
+...opy-assets-stac-task/payload_workflow.json: 4.58 KiB / 4.58 KiB ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 127.61 KiB/s 0s
+```
+
+Now modify the `secrets.yaml` and replace the `<REPLACE_WITH_BASE64_AWS_ACCESS_KEY>`, `<REPLACE_WITH_BASE64_AWS_SECRET_ACCESS_KEY>` and `<REPLACE_WITH_BASE64_AWS_REGION>` with the base64 encoded version of the AWS_ACCESS_KEY, AWS_SECRET_ACCESS_KEY and AWS_REGION of the iam user and bucket created on the [Pre-requisites section](#pre-requisites-to-running-argo-workflows-example).
+
+Create the kubernetes secret by executing:
+```
+kubectl apply -n default -f ./secret.yaml
+```
+
+Then modify the `<REPLACE_WITH_MINIO_HOST>:<REPLACE_WITH_MINIO_PORT>` variables inside the `workflow_copyassets_no_template.yaml` and the `workflow_copyassets_with_template.yaml`. If you're running this example from a minio created by the terraform stack, you should be able to replace `<REPLACE_WITH_MINIO_HOST>:<REPLACE_WITH_MINIO_PORT>` with `minio.default:9000`.
+
+Finally, run and watch the argo workflow task via:
+```
+argo submit -n default --watch ./workflow_copyassets_no_template.yaml
+```
+
+You should be able to see your workflow pod succeed in the terminal:
+<br></br>
+<p align="center">
+  <img src="../../images/argo_workflow_success.png" alt="Argo Workflow Success" width="776">
+</p>
+<br></br>
+
+And you should be able to see your asset S3 bucket populated with a tumbnail image:
+```
+aws s3 ls s3://copy-assets-stac-task-bucket/data/naip/tx_m_2609719_se_14_060_20201217/
+
+2023-08-17 10:00:08       9776 thumbnail.jpg
+```
+<br></br>
+
```

Comment:
should we also add a statement here on how to run the `workflow_copyassets_with_template.yaml `workflow?

`kubectl apply -f ./workflow-template.yaml
`
`argo submit -n default --watch ./workflow_copyassets_with_template.yaml `

Reply:
Added.